.travis.yml

```diff
@@ -1,12 +1,27 @@
 language: go
 sudo: required
 
+matrix:
+  include:
+  - name: amd64
+    os: linux
+    env: ARCH=amd64
+  - name: ppc64le
+    os: linux-ppc64le
+    env:
+      - ARCH=ppc64le
+      - DOCKERFILE=Dockerfile.ppc64le
+
 go:
   - "1.10"
+
 env:
   global:
   - PATH=$GOROOT/bin:$GOPATH/bin:$PATH
 
+before_install:
+  - sudo systemctl restart docker
```

Comment:
Why this is required?

Reply:
The CI for ppc64le failed to fetch filed when run the command to install the required packages, and restarting the docker service is the solution for it.
![Capture](https://user-images.githubusercontent.com/29833327/72871010-686dcd00-3cf2-11ea-8649-d6f5147fe6bb.PNG)